include/dav1d/headers.rs

```diff
@@ -2164,14 +2171,20 @@ pub struct Rav1dFrameHeader_restoration {
 impl From<Dav1dFrameHeader_restoration> for Rav1dFrameHeader_restoration {
     fn from(value: Dav1dFrameHeader_restoration) -> Self {
         let Dav1dFrameHeader_restoration { r#type, unit_size } = value;
-        Self { r#type, unit_size }
+        Self {
+            r#type: r#type.map(|e| Rav1dRestorationType::from_repr(e as usize).unwrap()),
+            unit_size,
+        }
     }
 }
```

Comment:
maybe this should be a `TryFrom` now?

Reply:
Yeah, that'd be good.

Reply:
Actually, leave it as an `.unwrap()`.  This is called when round-tripping across the C `DAV1D_API`, so there's not really an opportunity to bubble up an error.